pokedex/package.json

```diff
@@ -32,5 +32,5 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://localhost:3001"
+  "proxy": "http://localhost:3003"
 }
```

Comment:
Is there a reason why local host port number is changed?
it is not important though